pkgs/development/python-modules/xkcdpass/default.nix

```
@@ -19,6 +20,13 @@ buildPythonPackage rec {
     hash = "sha256-zEgC3tTQ6kwDovHPHRTvYndWVF79DpnAX454VDZiedE=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/redacted/XKCD-password-generator/commit/72d174a82822af1934c94de1b66fd956230142f5.patch";
```

```suggestion
      name = "fix-non-deterministic-test.patch";
      url = "https://github.com/redacted/XKCD-password-generator/commit/72d174a82822af1934c94de1b66fd956230142f5.patch";
```

Also, did you open a PR for this? We generally try to get patches upstream and make a note when they haven't yet been merged upstream..

---

Yes, I've just opened it: https://github.com/redacted/XKCD-password-generator/pull/158

(Has been out of time back then.)